backend/routes.py

```diff
@@ -0,0 +1,145 @@
+import uuid as u
+import copy
+import backend.pike as pike
+import tests.api_fixtures as api_mocks
+
+
+## Read calls
+# c.iii.1 (?) get available agents
+@pike.api.get("/agents")
+def get_public_agents() -> list[dict]:
+    """
+    Get a list of all public agent types.
+    """
+    return [api_mocks.mock_agent_one_skill, api_mocks.mock_agent_two_skills]
```

Comment:
why is it called `mock_agent_*_skill` instead of `mock_agent_*`? it sounds like it is a skill not agent